src/dev/build/tasks/os_packages/docker_generator/resources/base/bin/kibana-docker

```diff
@@ -306,7 +306,6 @@ kibana_vars=(
     xpack.ingestManager.registryUrl
     xpack.observability.annotations.index
     xpack.observability.unsafe.alertDetails.metrics.enabled
-    xpack.observability.unsafe.alertDetails.logs.enabled
```

Comment:
It's used at https://github.com/elastic/kibana/pull/172554/files#diff-53c98e8d9e2d61d5c9f37b003fc6de5c3f84570f7886cf4bdd89f596b32a15c9R73.

In a docker environment, without this instead of getting a deprecation warning the configuration will be ignored.  It's a wrapper around what environment variables are passed to bin/kibana.

Reply:
Are we okay with skipping the deprecation warning?

Reply:
The reason I didn't remove the config is to prevent kibana crashing if user has discovered and used config (following recommendation from [this document](https://docs.google.com/document/d/1J0MggFsHYXE3qkPnqw-GgrznbMuWizRdQdHHqyd3oVA/edit)).

The deprecation warning is nice to have but not a must. As long as it doesn't break kibana and silently ignores the config in docker environment, it should be fine.

Reply:
@jbudz I've added the feature flag back in `docker-kibana` ff0991caae5a07550c07f5f910d94e02f5e9a965. I will remove it when the config is removed as per your suggestion.

Reply:
@benakansara Was there another separate discussion about this topic? I am trying to understand why we need to pass this config instead of ignoring it and showing a warning.